blueprints/validator-test/index.js

```diff
@@ -47,5 +47,10 @@ module.exports = {
 
   _legacySrcPathOverride: function(file) {
     return path.resolve(this._filesPath(), file);
-  }
+  },
+
+  // workaround https://github.com/ember-cli/ember-cli/issues/5481
+  supportsAddon: function() {
+    return false;
```

Comment:
This is what's actually throwing the error - it's defined [here](https://github.com/ember-cli/ember-cli/blob/d8ba7dbaf77df0e7fe2daf1d153a1a5a60620143/lib/models/blueprint.js#L763-L773). It is defined as this:

```javascript
return !!this.files().join().match(/__root__/);
```

The `__root__` token is used in blueprints to allow being defined in either `app` or `addon` depending on context. However, this will always (and should always) return false for the test generator because, no matter the context, tests are always placed in `tests/` in an addon or app. By allowing this to short-circuit, we skip over the call to `files()` in the default definition which throws the error.